science/limesuite/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.0
+PortGroup           cxx11 1.1
+PortGroup           github 1.0
+PortGroup           wxWidgets 1.0
+
+name                limesuite
+version             v19.01.0
+categories          science comms
+platforms           darwin macosx
+license             Apache-2
+maintainers         {@ra1nb0w irh.it:rainbow} openmaintainer
+description         provides drivers and SDR application support for the LMS7002M RFIC
+long_description    Lime Suite is a collection of software supporting several hardware \
+    platforms including the LimeSDR, drivers for the LMS7002M transceiver RFIC, and other \
+    tools for developing with LMS7-based hardware.
+homepage            https://myriadrf.org/projects/lime-suite/
+github.setup        myriadrf LimeSuite 025ffa1a1026a6307b001ea2a6ffb9ab5c4e8d63
+
+checksums           rmd160  dbb56b3657d3e7fcdcda50e137400efd7f0d3417 \
+                    sha256  e7228c53d5fbe93a490d82fdcb94a54fb9a1d5cacc0bbf2c581609e859f3cd86 \
+                    size    5347698
+
+depends_lib-append  port:libusb \
+                    port:sqlite3
+
+configure.args-append \
+    -DDOWNLOAD_IMAGES=OFF \
+    -DENABLE_STREAM=ON \
+    -DENABLE_GUI=OFF \
+    -DENABLE_NOVENARF7=OFF \
+    -DENABLE_SOAPY_LMS7=OFF\
+    -DENABLE_OCTAVE=OFF
+
+# need 3.2 to enable dark mode in mojave
+variant wxgui description {Enable LimeSuiteGUI} {
+    wxWidgets.use           wxWidgets-3.2
+    depends_lib-append      port:${wxWidgets.port}
+    configure.args-delete   -DENABLE_GUI=OFF
+    configure.args-append   -DENABLE_GUI=ON
+}
+
+variant soapy description {Add Soapy support} {
+    depends_lib-append      port:SoapySDR
+    configure.args-delete   -DENABLE_SOAPY_LMS7=OFF
+    configure.args-append   -DENABLE_SOAPY_LMS7=ON
+}
+
+variant download_images description {Download and install firmware/gatware images} {
+    configure.args-delete   -DENABLE_STREAM=OFF
+    configure.args-append   -DENABLE_STREAM=ON
+}
+
+variant octave description {Enable Octave support} {
+    depends_lib-append      path:bin/octave:octave
+    depends_skip_archcheck  octave octave-devel
+    configure.args-delete  -DENABLE_OCTAVE=OFF
+    configure.args-append  -DENABLE_OCTAVE=ON
+}
+
+default_variants +wxgui +soapy
+
+# do VPATH (out of source tree) build
+cmake.out_of_source yes
```

Comment:
This line is redundant in the cmake 1.1 portgroup and can be removed if you switch to that.

Reply:
Nice. thanks